getting-started/data-scientists.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Data scientists
+description: Get started with Coder as a data scientist.
+---
+
+This article will walk you through the process of getting started with a Coder
+workspace capable of supporting data science projects. You'll learn how to:
+
+- Connect Coder to your Git provider;
+- Create a workspace with Jupyter and other data science packages present;
+- Add a sample project to your workspace, specifically
+  [one in Jupyter notebook form using IMDB movie data](https://github.com/khorne3/data-science-imdb-sample)
+- Create a dev URL and preview changes to your project.
+
+## Prerequisites
+
+This guide assumes that you have a Coder deployment available to you and that
+you have the credentials needed to access the deployment.
+
+## Step 1: Log in and connect Coder to your Git provider
+
+In this step, you'll log into Coder and connect and authenticate with your Git
```

Comment:
the "log into Coder and connect and authenticate" seems awkward, due to repeating _and_. Maybe change the first one to "then"? And/or remove the _authenticate_: "In this step, you'll log into Coder, then link your Coder account with your Git provider"